submissions/views.py

```diff
@@ -0,0 +1,18 @@
+from django.shortcuts import render
+from hackathon.models import HackProject
+from .forms import SubmissionForm
+
+def submit(request):
+    if request.method == 'POST':  # If the form has been submitted...
+        form_data = request.POST
+        submission_form = SubmissionForm(form_data)
+        if submission_form.is_valid():
+            submission = submission_form.save()
+            return render(request, 'submissions/success.html')
+    submission_form = SubmissionForm()
+
+    context = {
```

Comment:
Here I am not sure you need to create an extra context dict with only one key, I would just pass this directly into the render function.